app/controllers/catalog_controller.rb

```diff
@@ -3,11 +3,11 @@
 ##
 # Blacklight controller providing search and discovery features
 class CatalogController < ApplicationController
-  include BlacklightRangeLimit::ControllerOverride
   include BlacklightAdvancedSearch::Controller
   helper Openseadragon::OpenseadragonHelper
 
   include Blacklight::Catalog
+  include BlacklightRangeLimit::ControllerOverride
```

Comment:
👍 are other folks going to need to ensure that this include comes after `Blacklight::Catalog`?  Just wondering if we should spin off a ticket about updating the generator to be more intelligent about where it's injecting this include.

Reply:
Great idea, added https://github.com/projectblacklight/blacklight_range_limit/issues/134